### ISSUE TITLE
[Bug]: 6.0.37 categorizes TV show with 1 animated episode as animation even though it's not

### ISSUE THREAD

**Issue by Pentaphon** [nova-video-player/aos-AVP#549]
### Problem description

A minor issue with an edge case. Not high priority but something to tweak in the future. 

This mostly live-action show, Amazing Stories (1985), had 1 animated episode aka S02E16, so on TMDB it is categorized with the following tags:  Sci-Fi & Fantasy, Drama, Comedy, Family, Animation, Mystery, Crime. Since Animation is in there, Nova puts it under the animation row, but it should just go under the TV series row. 

 https://www.themoviedb.org/tv/1026-amazing-stories

### Steps to reproduce the issue

1) Add the filenames from this show to simulate actually having it. 
2) See that Nova categorizes Amazing Stories (1985) as an animation. See screenshot. 

![as](https://user-images.githubusercontent.com/49374058/150634846-c4742493-55c0-4424-88fd-59389bb8a6b5.jpg)

### Expected behavior

Amazing Stories (1985) should be categorized as a TV show because the show is mostly live-action so a bit of tweaking to the scraping logic is needed to detect it as a TV show and as not as Animation. 

### Your phone/tablet/androidTV model

FireTV 4K Stick

### Operating system version

FireOS 7

### Application version and app store

6.0.37 from Github

### Additional system information

Not relevant

### Debug logs

Not necessary; easily reproducible.

**Comment by arikoza** [comment]
The player is here?  you need to fix it on tmdb, there is a genre "Animation"

**Comment by courville** [comment]
OK I tried to reproduce with the animated s02e16 and scraping in french does not end up in animation for some reason. Will try in english soon.

**Comment by Pentaphon** [comment]
> OK I tried to reproduce with the animated s02e16

It gets categorized as animation even if you don't have the animated s02e16 on your share. I think the issue is that if Nova sees that TMDB has "animation" as a genre for a movie or tv show, it just puts it in the animation row. This issue might not be fixable unless TMDB has primary, secondary and tertiary genres and Nova **_only_** categorizes a movie or TV show as animation if animation is a primary genre. 

Edit: somebody named "arikoza" removed "animation" from the list of genres for this show on TMDB and it wasn't me. Whoever did this, you may not want to change info on TMDB in case somebody reports you to a mod over there. 

@courville there is an "order" when you edit/add genres on TMDB. Perhaps the solution is simply to **_only_** categorize a show as animation if the 1st order genre is animation rather than the 2nd, 3rd, 4th, etc. 

![Screenshot 2022-01-26 at 15-29-37 Amazing Stories](https://user-images.githubusercontent.com/49374058/151264466-69c7432c-92f8-4e74-ac81-5984df06ef8f.png)
https://www.themoviedb.org/tv/1026-amazing-stories

**Comment by courville** [comment]
OK this explains why I cannot reproduce it anymore.
Since order of the genres is potentially not hierarchic on tmdb and could result in even more side effects, I propose to stick with tmdb being the source of truth and not implement work arounds for now.

**Comment by Pentaphon** [comment]
> Since order of the genres is potentially not hierarchic on tmdb and could result in even more side effects

Yeah I couldn't figure out whether the genre order does anything for the software API but I may not have dug deep enough to get the right answer. Also, I just realized @arikoza is the guy who is also in this issue. I guess his edit will have to do for now, but again, I don't recommend anybody edit TMDB in case the tmdb mods think you are making bad edits over there since technically, this show did have one animated episode, making the animation genre valid. I would just have put the animation genre at the lowest order.

**Comment by Pentaphon** [comment]
>I propose to stick with tmdb being the source of truth and not implement work arounds for now.

Closing
